client-metrics/src/main/scala/org/http4s/client/metrics/core/MetricsOps.scala

```diff
@@ -4,9 +4,8 @@ import cats.effect.Sync
 import org.http4s.Status
 
 /**
-  * Trait for objects capable of writing metrics to a metrics registry
+  * Describes an algebra capable of writing metrics to a metrics registry
```

Comment:
@SystemFw likes algebras to have an implicit summoner with explicit instances, and I'm coming around to that view myself.

How about also:

```scala
object MetricsOps {
  def apply[F[_]](implicit ev: MetricsOps[F]) = ev
}
```

Reply:
What is the advantage of this? Having the actual implementation of MetricsOps being defined by the implicits in scope?

Reply:
I'm trying to get my head around this, but I still not understanding very well.
How would this work?
```
implicit val dropwizardMetricsOps = Dropwizard(registry, { _: Request[F] => Some("classifier")})
val metricsOps = MetricsOps()
```
I doesn't make much sense to me right now...  thinking

Reply:
It makes sense for a lot of algebra interpreters, but I think you're right.  It doesn't make much sense here.  Ignore this one.